src/index.html

```diff
@@ -6,8 +6,62 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+
+      <img class="card__image"
+           src="./images/imac.jpeg"
+           alt="A1419 IMac">
```

Comment:
apply code styling recommendations from a [checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
```suggestion
      <img 
        class="card__image"
        src="./images/imac.jpeg"
        alt="A1419 IMac"
      >
```